config/webpack.config.dev.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  devtool: '#source-map',
+  externals: ['cozy'],
+  plugins: [
+    new webpack.DefinePlugin({
+      __SERVER__: JSON.stringify('http://app.cozy.local'),
```

Comment:
Actually I copy-pasted from another cozy app, maybe cozy-photos-v3. I am updating right now to `cozy.tools`.